source/linux/Foundational_Components/Kernel/Kernel_Drivers/Display/DSS7.rst

```diff
@@ -362,7 +362,10 @@ Device Tree Node
 Documentation for tidss device tree node and its properties can be found in linux kernel device tree bindings in below directory
 ``Documentation/devicetree/bindings/display/ti/``. Seperate binding files are present for different version of the DSS controller.
 
+.. ifconfig:: CONFIG_part_variant in ('J721S2')
 
+        - Pin designated as HPD for DP should not be used for other purpose, irrespective of whether it is connected as HPD or not.
```

Comment:
> The pin designated as HPD for DP should not be used for other purpose, irrespective of whether it is connected to HPD.

Indentation here is also a bit odd. RST standard is 3 spaces. This file is currently using 4 spaces, but this line is using 8.